grl/run.py

```diff
@@ -9,8 +9,9 @@
 from .mdp import MDP, AbstractMDP
 from .mc import mc
 from .policy_eval import PolicyEval
+from .grad import do_grad
 
-def run_algos(spec, no_gamma, n_random_policies, n_steps, max_rollout_steps):
+def run_algos(spec, no_gamma, n_random_policies, grad, n_steps, max_rollout_steps):
```

Comment:
Can you change `grad` to `use_grad` to avoid confusion with `jax.grad`?

Reply:
Done